devices/CMakeLists.txt

```diff
@@ -3,4 +3,7 @@
 # GNU Lesser General Public License v2.1 or any later version.
 
 add_subdirectory(baseEstimatorV1)
-
+add_subdirectory(wholeBodyDynamics)
+add_subdirectory(virtualAnalogClient)
```

Comment:
I suppose these devices were added to YARP during these years. Could we add a small note or open an issue as to refactor out these devices out of this repository? And add the necessary documentation to use the YARP devices instead.

Reply:
Unfortunately I never actually migrated this devices in YARP.

Reply:
The device `genericSensorClient` though? Is it negative also for that?

Reply:
Unfortunately that as well.

Reply:
Ah, I see. My bad, then.